src/services/listen-service/meme.ts

```diff
@@ -108,7 +108,7 @@ export const meme = (
   { content }: BotCommand,
```

Comment:
this is good, but we can do better and make this a lil more terse. Also, can't append arrays to `FormData`